YoutubeExplode/Bridge/Extractors/VideoWatchPageExtractor.cs

```diff
@@ -89,6 +88,8 @@ internal partial class VideoWatchPageExtractor
                 .Pipe(j => new PlayerResponseExtractor(j))
         );
 
+        public static PlayerResponseExtractor? TryGetPlayerResponse(string json) => String.IsNullOrWhiteSpace(json) ? null : new PlayerResponseExtractor(Json.Parse(json));
```

Comment:
Can `json` ever be null or whitespace here? It seems to be passed from an HTTP request where it's reasonable to assume it's always going to be non-whitespace.